opentracing-spring-cloud/src/main/java/io/opentracing/contrib/spring/cloud/zuul/TracePostZuulFilter.java

```diff
@@ -0,0 +1,72 @@
+package io.opentracing.contrib.spring.cloud.zuul;
+
+import com.netflix.zuul.ZuulFilter;
+import com.netflix.zuul.context.RequestContext;
+import io.opentracing.Span;
+import io.opentracing.tag.Tags;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TracePostZuulFilter extends ZuulFilter {
+    static final String ROUTE_HOST_TAG = "route.host";
```

Comment:
There is no formatting check, but could you please update files to use 2 spaces instead of 4.